test/validation/api/timer/timer.c

```diff
@@ -376,8 +377,9 @@ static void timer_test_capa_allsrc(void)
 	CU_ASSERT_FATAL(ODP_CLOCK_EXT == ODP_CLOCK_SRC_1);
 #endif
 
+	CU_ASSERT(odp_timer_capability(ODP_CLOCK_DEFAULT, &capa) == 0);
```

Comment:
Please merge this patch to the first patch, since that modifies overall how different clock sources are tested.

Reply:
In v5.